install/kibana-install.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: T.H. (ELKozel)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y wget
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y ca-certificates
+$STD apt-get install -y apt-transport-https
+$STD apt-get install -y gnupg
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up Elastic Repository"
+mkdir -p /etc/apt/keyrings
+wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | gpg --dearmor -o /usr/share/keyrings/elasticsearch-keyring.gpg
+echo "deb [signed-by=/usr/share/keyrings/elasticsearch-keyring.gpg] https://artifacts.elastic.co/packages/8.x/apt stable main" >/etc/apt/sources.list.d/elastic-8.x.list
+msg_ok "Set up Elastic Repository"
+
+msg_info "Installing Kibana"
+$STD apt-get update
+$STD apt-get install kibana
+msg_ok "Installed Kibana"
+
+msg_info "Creating Service"
+cat <<EOF >/etc/systemd/system/Kibana.service
+[Unit]
+Description=Kibana
+Documentation=https://www.elastic.co/guide/en/kibana/8.15/index.html
+Wants=network-online.target
+After=network-online.target
+[Service]
+Type=simple
+User=kibana
```

Comment:
Is the User & Group setup during the install?

Reply:
Yes, just checked it, this is the `/etc/passwd` after a clean install:
```
root@kibana:~# cat /etc/passwd
root:x:0:0:root:/root:/bin/bash
daemon:x:1:1:daemon:/usr/sbin:/usr/sbin/nologin
bin:x:2:2:bin:/bin:/usr/sbin/nologin
sys:x:3:3:sys:/dev:/usr/sbin/nologin
sync:x:4:65534:sync:/bin:/bin/sync
games:x:5:60:games:/usr/games:/usr/sbin/nologin
man:x:6:12:man:/var/cache/man:/usr/sbin/nologin
lp:x:7:7:lp:/var/spool/lpd:/usr/sbin/nologin
mail:x:8:8:mail:/var/mail:/usr/sbin/nologin
news:x:9:9:news:/var/spool/news:/usr/sbin/nologin
uucp:x:10:10:uucp:/var/spool/uucp:/usr/sbin/nologin
proxy:x:13:13:proxy:/bin:/usr/sbin/nologin
www-data:x:33:33:www-data:/var/www:/usr/sbin/nologin
backup:x:34:34:backup:/var/backups:/usr/sbin/nologin
list:x:38:38:Mailing List Manager:/var/list:/usr/sbin/nologin
irc:x:39:39:ircd:/run/ircd:/usr/sbin/nologin
_apt:x:42:65534::/nonexistent:/usr/sbin/nologin
nobody:x:65534:65534:nobody:/nonexistent:/usr/sbin/nologin
systemd-network:x:998:998:systemd Network Management:/:/usr/sbin/nologin
postfix:x:100:109::/var/spool/postfix:/usr/sbin/nologin
sshd:x:101:65534::/run/sshd:/usr/sbin/nologin
systemd-timesync:x:997:997:systemd Time Synchronization:/:/usr/sbin/nologin
messagebus:x:102:111::/nonexistent:/usr/sbin/nologin
kibana:x:103:112::/nonexistent:/bin/false
```